Assets/Tests/Play/VimeoRecorderPlayTest.cs

```diff
@@ -100,6 +100,21 @@ public IEnumerator Can_Record_Video_From_MainCamera_With_Valid_Token()
         }
     }
 
+    [UnityTest]
+    public IEnumerator Can_Record_Video_Without_Uploading()
+    {
+        recorder.videoName = "Screen Test " + recorder.videoName;
+        recorder.defaultVideoInput = VideoInputType.Screen;
+        recorder.SignIn(VALID_RECORDING_TOKEN);
+        recorder.autoUpload = false;
```

Comment:
There should be a unit test that simply tests that `autoUpload` defaults to true.

Reply:
Added this and tests to all default user facing values on the Vimeo Recorder GUI - https://github.com/vimeo/vimeo-unity-sdk/commit/3f52dd98475c9ad763be2b22d6d264d2ad3f7259